pkg/seccomp/seccomp_test_victim_arm64.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2020 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Test binary used to test that seccomp filters are properly constructed and
+// indeed kill the process on violation.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"syscall"
+
+	"gvisor.dev/gvisor/pkg/seccomp"
+)
+
+func main() {
+	dieFlag := flag.Bool("die", false, "trips over the filter if true")
+	flag.Parse()
+
+	syscalls := seccomp.SyscallRules{
```

Comment:
This file duplicates 95% of pkg/seccomp/seccomp_test_victim_amd64.go. I think it is better to wrap x86 syscalls with "ifdef __x86_64__".

Reply:
> This file duplicates 95% of pkg/seccomp/seccomp_test_victim_amd64.go. I think it is better to wrap x86 syscalls with "ifdef **x86_64**".

Hi, Andrei, I did some research on howto use "ifdef" in golang script. The only way I found is by using CGO. I am not sure if it is the approach you prefered. Or can you give me some suggestion on it?

Reply:
I am sorry, I forgot to switch from C to golang to review this change :). Let me thing what we can do this case.

Reply:
> I am sorry, I forgot to switch from C to golang to review this change :). Let me thing what we can do this case.

Ok, thanks Andrei.

Reply:
What do you think about something like this https://github.com/avagin/gvisor/commit/0869d359a4fc6171131fac7e77b9bf9da51be510

Reply:
> What do you think about something like this [avagin@0869d35](https://github.com/avagin/gvisor/commit/0869d359a4fc6171131fac7e77b9bf9da51be510)

Good solution!